src/lwc/rdActiveSchedule/rdActiveSchedule.js

```diff
@@ -1,8 +1,8 @@
-import lblScheduleTitle from '@salesforce/label/c.RD2_ScheduleLWCTitle';
-import lblCurrentSchedule from '@salesforce/label/c.RD2_CurrentScheduleTitle';
-import lblFutureSchedule from '@salesforce/label/c.RD2_ScheduleLWCFutureSchedule';
-import lblRecordIcon from '@salesforce/label/c.AssistiveTextRecordIcon';
-import lblNone from '@salesforce/label/c.stgLabelFieldValueNone';
+import scheduleTitle from '@salesforce/label/c.RD2_ScheduleLWCTitle';
```

Comment:
Not sure I understand the reason for these changes.  Are you just trying to get away from specifying type info in a name?  If so, seems fine to me.

Reply:
since all the labels are stored in the Label json file in the javascript file. I find the `lbl` prefix on the label itself unnecessary. Instead of having `Labels.lblScheduleTitle` in the html, it will not become `Labels.scheduleTitle` which is easier to read.